The Form Widget (#2374)
umbraco/UmbracoDocs: Add-ons/Starter-Kit-TEMP/Widgets/index.md

- Buttons

![movePage.jpg](images/The-Latest-Form-Feed-Widget1.png)

### The Umbraco Form Widget

![movePage.jpg](images/The-Umbraco-Form-Widget.png)

> The Umbraco Form widget gives you the ability to add a form to your site, it is important to note that this form needs to have been made before this.

Reviewer: ```suggestion
The Umbraco Form widget gives you the ability to add a custom form to your site. It is important to note that a custom form needs to have been made using Umbraco Forms before this widget can be used.
```

> You can create a form by selecting the forms link in the top menu in your backoffice. 

Reviewer: ```suggestion
You can create a form by selecting the Forms section in the top menu in your backoffice. 
```

> You will have some settings to personalize your form they are as follows:

Reviewer: ```suggestion
You will have some settings to personalize how your form is presented through the widget. The settings are as follows:
```

- Pre Heading
- Heading
- Text
- Background Image
- Background Color
- Text Color

> - Form ( this is where you add the form that you have already collected)

Reviewer: ```suggestion
- Form (this is where you add the form that you have already created)
```
